### PR TITLE
stability: add fault-trigger client

### PR DESCRIPTION
**Pull request by cwen0**
@weekface @zyguan @xiaojingchen PTAL

### CODE REVIEW

File: tests/pkg/fault-trigger/client/client.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+	StopKubelet() error
+}
+
+// client is used to communicate with the fault-trigger
+type client struct {
+	cfg     Config
+	httpCli *http.Client
+}
+
+// Config defines for schrodinger client
```
Comment (weekface) [inline tests/pkg/fault-trigger/client/client.go]: Remove `schrodinger` literal?

File: tests/pkg/fault-trigger/client/client.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+		return err
+	}
+
+	vmName := vm.Name
+	if len(vmName) == 0 {
+		vmName = vm.IP
+	}
+
+	url := util.GenURL(fmt.Sprintf("%s/%s/vm/%s/start", c.cfg.Addr, api.APIPrefix, vmName))
+	if _, err := c.get(url); err != nil {
```
Comment (weekface) [inline tests/pkg/fault-trigger/client/client.go]: Do all operations use the HTTP `Get` method?

**Comment by cwen0** [comment]
@weekface PTAL again, thanks!

File: tests/pkg/fault-trigger/client/client.go (hunk 2 above)
Comment (cwen0) [reply tests/pkg/fault-trigger/client/client.go]: If the operation changes the state, I will replace the`Get` with `Post` in a sigle pr.

**Pull request merged**